data/json/items/magazines.json

```diff
@@ -8,12 +8,62 @@
         "description" : "A standard capacity magazine for use with STANAG compatible rifles.",
         "material" : "steel",
         "volume" : 2,
-        "weight" : 110,
-        "to_hit" : -2,
+        "weight" : 360,
         "ammo_type" : "223",
         "capacity" : 30,
-        "reliability" : 10,
-        "reload_time" : 30
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ]
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "stanag10",
+        "price" : 5000,
+        "name" : "Ruger 223 magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A compact 10-round magazine commonly used with semi-automatic STANAG compatible rifles.",
+        "material" : "steel",
+        "volume" : 1,
+        "weight" : 100,
+        "ammo_type" : "223",
+        "capacity" : 10,
+        "reliability" : 9,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ]
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "stanag50",
+        "price" : 9000,
+        "name" : "STANAG drum magazine",
```

Comment:
For ammo belts in .223. 200 rounds seems about right.

Regarding the 50 round capacity. I suppose you could name is ``Extended STANAG magazine`` or something. Or make it a casket mag, that still kinda works for 50 rounds.